lib/src/widgets/movie_vote_average.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import '/../utils/movie_dimensions.dart';
+
+class MovieVoteAverage extends StatelessWidget {
+  final int starCount;
+  final double voteAverage;
+
+  const MovieVoteAverage({
+    Key? key,
+    this.starCount = MovieDimensions.starsAmount,
+    required this.voteAverage,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        Text(
+          voteAverage.toString(),
+          style: TextStyle(
+            color: Colors.yellow,
+            fontSize: MovieDimensions.titleFontSize,
+          ),
+        ),
+        SizedBox(
```

Comment:
This can be const, like Javi mention in the video

Reply:
Now is a const type, thank you